apps/coscrad-frontend/src/components/resources/photographs/photograph-detail.thumbnail.presenter.tsx

```diff
@@ -0,0 +1,26 @@
+import { IDetailQueryResult, IPhotographViewModel, ResourceType } from '@coscrad/api-interfaces';
+import { routes } from 'apps/coscrad-frontend/src/app/routes/routes';
+import { Link } from 'react-router-dom';
+import './photograph-detail.thumbnail.presenter.css';
+
+export const PhotographDetailThumbnailPresenter = ({
+    data: photograph,
+}: IDetailQueryResult<IPhotographViewModel>): JSX.Element => {
+    const location = `/${routes.resources.ofType(ResourceType.photograph).detail(photograph.id)}`;
```

Comment:
Let's not call this location. Let's call this something like `pathToDetailPageForThisPhotograph`.

Reply:
Done